2024/01/02 17:49:44 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: geoservice.by, request: "GET /swagger HTTP/1.1", upstream: "http://192.168.48.20:8080/swagger", host: "localhost:8080"
2024/01/02 17:49:44 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: geoservice.by, request: "GET /swagger HTTP/1.1", upstream: "http://192.168.48.18:8080/swagger", host: "localhost:8080"
2024/01/02 17:49:44 [error] 7#7: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: geoservice.by, request: "GET /swagger HTTP/1.1", upstream: "http://192.168.48.19:8080/swagger", host: "localhost:8080"
2024/01/02 17:49:45 [error] 7#7: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: geoservice.by, request: "GET /swagger HTTP/1.1", upstream: "http://proxy_services/swagger", host: "localhost:8080"
2024/01/02 17:49:45 [error] 7#7: *1 no live upstreams while connecting to upstream, client: 192.168.48.1, server: geoservice.by, request: "GET /swagger HTTP/1.1", upstream: "http://proxy_services/swagger", host: "localhost:8080"
